src/main/java/io/xpring/ilp/DefaultIlpClient.java

```diff
@@ -78,16 +77,16 @@ public AccountBalance getBalance(final String accountId, final String accessToke
             // Convert protobuf response to AccountBalanceResponse
             return AccountBalance.from(response);
         } catch (StatusRuntimeException statusRuntimeException) {
-            throw new XpringException(String.format("Unable to get balance for account %s.  %s", accountId, statusRuntimeException.getStatus()));
+            throw new IlpException(IlpExceptionType.UNKNOWN, String.format("Unable to get balance for account %s.  %s", accountId, statusRuntimeException.getStatus()));
```

Comment:
This is good for now.  I will do something similar to what I am doing in the JS and Swift PRs to translate StatusRuntimeExceptions to IlpExceptions

Reply:
Yup. I figured I'd give you a start and then you could just make a breaking change when required.